src/snapred/backend/dao/reduction/ReductionRecord.py

```diff
@@ -17,35 +16,19 @@ class ReductionRecord(BaseModel):
     calibration, normalization, and pixel grouping details.
     """
 
-    runNumbers: List[str]
+    # Reduction has distinct registration attributes from Calibration and Normalization.
+    #   for this reason, this class is not derived from 'Record',
+    #   and does not include a 'CalculationParameters' instance.
     useLiteMode: bool
+    timestamp: float = Field(frozen=True, default=None)
+
+    # specific to reduction records
+    runNumber: str
```

Comment:
So the change from a list of run numbers to a single run number input was made as this release only allows for single run number reductions at a time? Or am I confusing myself?

Reply:
We still allow a _batch_ of run numbers, but they all need to be from the same state.  It's just that the `ReductionRecord` and `ReductionRequest` and etc. happen _one_ run at a time.